app/component/RouteStopListContainer.js

```diff
@@ -96,6 +96,11 @@ class RouteStopListContainer extends React.PureComponent {
           first={i === 0}
           className={rowClassName}
           displayNextDeparture={this.context.config.displayNextDeparture}
+          prevVehicleDeparture={
+            i > 0 && stops[i - 1].stopTimesForPattern[0]
```

Comment:
This expression is quite complicated. Can you remove the duplication of `stops[i - 1].stopTimesForPattern[0]`.